rebar.config

```diff
@@ -1,7 +1,7 @@
 {deps, [
     {lager, {git, "https://github.com/basho/lager", {tag, "3.2.4"}}},
     {jsx, {git, "https://github.com/talentdeficit/jsx.git", {tag, "master"}}},
-    {erldns, {git, "https://github.com/dcos/erl-dns.git", "deprecate-records_by_type"}},
+    {erldns, {git, "https://github.com/dcos/erl-dns.git", master}},
```

Comment:
Is there recommended style for rebar.config? I tried to use the style from this file. Nobody used `{branch, "master"}` here. Should I change it?